src/planner/ddl_plan.go

```diff
@@ -66,162 +65,213 @@ func NewDDLPlan(log *xlog.Log, database string, query string, node *sqlparser.DD
 	}
 }
 
-// Build used to build DDL distributed querys.
-// sqlparser.DDL is a simple grammar ast, it just parses database and table name in the prefix.
-func (p *DDLPlan) Build() error {
+// checkUnsupportedOperations used to check if we do unsupported operations if shardtype is HASH.
```

Comment:
comment is  confuse.

Reply:
Thanks, I had changed first if to whether, and second if to when.